src/test/java/org/sonarsource/sonarlint/ls/mediumtests/AbstractLanguageServerMediumTests.java

```diff
@@ -463,6 +464,11 @@ public void showRuleDescription(ShowRuleDescriptionParams params) {
     public void showHotspot(HotspotDetailsDto h) {
     }
 
+    @Override
+    public void showIssue(ShowIssueParams issue) {
+
```

Comment:
This could store the params into a collection that would allow for assertions in a medium test.